src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+
+- Using 13,000,000 total ETH staked, the rate of ETH issuance is ~1600 ETH/day ([See source](https://ultrasound.money/))
+- Results in **~584,000** ETH issued in a year
+- Currently inflating at **~0.49%** (584K per year / 119.3M total)
+
+<InfoBanner>
+<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)
+
+**~89.4%** of the issuance is going to miners on the execution layer (4.13 / 4.62 \* 100)
+
+**~10.6%** is being issued to stakers on the consensus layer (0.49 / 4.62 \* 100)
+</InfoBanner>
+
+## Post-merge {#post-merge}
+
+### Execution layer issuance {#el-issuance-post-merge}
+
+Will be zero. Proof-of-work will no longer be valid under the rules of consensus, and all execution layer activity will be included in "beacon blocks" which are published and attested to by proof-of-stake validators.
+
+### Consensus layer issuance {#cl-issuance-post-merge}
+
+Will continue as it was, with small rewards for attesting-to and proposing blocks. These rewards will continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
+
+When validator withdrawals are enabled, stakers will be incentivized to remove their _earnings/rewards (balance over 32)_ as these funds are otherwise not contributing to their stake weight (which maxes as 32).
+
+Stakers may also choose to exit and withdraw their entire validator balance, but it is important to remember that full validator exits are rate limited. Depending on the total ETH staked at the time, only 6 validators may exit in a given epoch (6.4 minute period). This decreases to as low as 4 as more validators leave, to intentionally prevent the ability for large amounts of staked ETH to leave at once.
+
+### Post-merge inflation breakdown {#post-merge-inflation-breakdown}
+
+- Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
+- Execution layer issuance: **0**
+- Consensus layer issuance: Same as above, **~0.49%** annual issuance rate (with 13 million ETH staked)
+- Total annual issuance rate: **~0.49%**
+
+<InfoBanner>
+Total annual issuance rate: <strong>~0.49%</strong>
+Net reduction in annual ETH issuance: <strong>~89.4%</strong> (0.49% / 4.62% * 100)
+</InfoBanner>
+
+## <Emoji text=":fire:" size="1" /> The burn {#the-burn}
+
+This flip side of ETH issuance is the rate at which ETH is burned. To execute a transaction on Ethereum, a minimum fee (known as a `base fee`) must be paid, which fluctuates depending on network activity in the moment. This fee is paid in ETH, is _required_ for the transaction to be considered valid, and is _burned_ in the process, removing it from circulation.
```

Comment:
Wasn't clear on the meaning of 'this flip side', I think this is what was meant?

```suggestion
The other thing to consider about ETH issuance is the rate at which ETH is burned. For a transaction to execute on Ethereum, a minimum fee (known as a `base fee`) must be paid, which fluctuates depending on network activity. The fee is paid in ETH and is _required_ for the transaction to be considered valid. The fee gets _burned_ during the transaction process, removing it from circulation.
```

Reply:
Kept most updates here, adjusted to "The opposite force to ETH issuance" since I already introduced the idea of opposing forces at the beginning.